test/drenv/__main__.py

```diff
@@ -124,6 +124,12 @@ def handle_termination_signal(signo, frame):
     sys.exit(1)
 
 
+def cmd_setup(env, args):
```

Comment:
Should we document this step in the user-quick-start.md?

Reply:
Note that in this commit message you setup `$ drenv setup envs/regional-dr.yaml`
but end up starting `$ drenv start envs/kubevirt.yaml -v`

Reply:
> Should we document this step in the user-quick-start.md?

Yes we must, thanks!

Reply:
> Note that in this commit message you setup `$ drenv setup envs/regional-dr.yaml` but end up starting `$ drenv start envs/kubevirt.yaml -v`

The issue is that command line parsing is too simplistic, so the envfile is required for every command. Like "clear", the new "setup" command requires an enf file but does not use it.

I'll fix this in the next version.

Reply:
> > Should we document this step in the user-quick-start.md?
> 
> Yes we must, thanks!

Current version setup minikube by default when creating virtual env (make venv) so this change is transparent for users and developers.